include/dlaf/lapack_tile.tpp

```diff
@@ -8,6 +8,30 @@
 // SPDX-License-Identifier: BSD-3-Clause
 //
 
+#include "dlaf/common/assert.h"
+
+namespace dlaf {
+namespace tile {
+
+template <class T>
+void lacpy(const Tile<const T, Device::CPU>& a, const Tile<T, Device::CPU>& b) {
+  DLAF_ASSERT_HEAVY(a.size() == b.size(),
+                    "Source and destination tile must have the same size. A=", a.size(),
+                    " B=", b.size());
+
+  SizeType m = a.size().rows();
+  SizeType n = a.size().cols();
+
+  lapack::lacpy(lapack::MatrixType::General, m, n, a.ptr(), a.ld(), b.ptr(), b.ld());
+}
+
+template <class T, Device device>
+void potrf(blas::Uplo uplo, const Tile<T, device>& a) {
+  auto info = potrfInfo(uplo, a);
+  if (info != 0)
+    throw std::runtime_error("Error: POTRF: A is not positive definite.");
```

Comment:
This task is going on in another place. I would leave it for the dedicated PR, since here I've just re-positioned it in the file (it would ease the merge later).

Do you agree?

Reply:
Yeah!